helper/schema/schema_test.go

```diff
@@ -1886,6 +1889,60 @@ func TestSchemaMap_Diff(t *testing.T) {
 
 			Err: false,
 		},
+
+		// #47 - https://github.com/hashicorp/terraform/issues/824
```

Comment:
47 is just the index number. Its just a jank historical thing that the test cases are a slice and the error says what index errored. We put it there for fast lookup. Newer style tests using a `map[string]TestCase` style where we use a descriptor, but converting 48 (0-based) tests to that is annoying, and it really doesn't affect usability that much as long as we keep adding new indexes.